src/styles/components/_c-card.scss

```diff
@@ -0,0 +1,39 @@
+.c-card {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  width: 250px;
```

Comment:
Absolute width on columns without a grid will cause the cards to overlap I think. See gif attached. Think this might affect the text and icons also. A design discussion for Ari also.
![fw-grid-overlap](https://cloud.githubusercontent.com/assets/20288774/26634050/a89799a0-4615-11e7-9f63-e47e574f80e2.gif)